### ISSUE TITLE
item's "_id" property keeps changing

### ISSUE THREAD

**Issue by nickojs** [osrsbox/osrsbox-api#20]
I'm implementing this API in a portfolio project, but it looks like the item's _id property keeps changing.

In my project, the user can search for items and then save them into his inventory. After a while (not sure if its a fixed amount of time, or random), when fetching saved items - by the _id property - I receive an empty _items array. By searching the same Item again, I realize the _id has changed. 

Is there any immutable property that I can use to save the items?

Also, congrats on the amazing project!

**Comment by osrsbox** [comment]
Hey @nickojs - instead of using the `_id` property, I would recommend using the `id` property (without the underscore prefix) to identify item "uniqueness". The `id` property is what the game uses to identify items, so it will not change (FYI - sometimes, but very rarely, JaGeX will change an item ID). The `_id` property that you are currently using is the MongoDB ObjectID property (see: https://docs.mongodb.com/manual/reference/method/ObjectId/). Like you said... this changes in my API for the same item. This is because I recreate the database every time there is an update which changes this value - which creates a new ObjectID. I could keep the `_id` property the same, but the update process (for me) would be slower and much more complex.

TLDR; Use `id` not `_id`

Hope this helps. And feel free to reopen this issue, or reply if it doesn't. And glad the API project can be of use to you! Good luck with your portfolio project!
